src/test/java/org/broadinstitute/hellbender/tools/spark/BaseRecalibratorSparkIntegrationTest.java

```diff
@@ -67,22 +67,22 @@ private String getCloudInputs() {
         final String localResources =  getResourceDir();
         final String hg19Ref = "EMWV_ZfLxrDY-wE";
         final String GRCh37Ref = ReferenceAPISource.URL_PREFIX + ReferenceAPISource.GRCH37_REF_ID;
-        final String HiSeqBam = localResources + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam";
-        final String dbSNPb37 =  getResourceDir() + "dbsnp_132.b37.excluding_sites_after_129.chr17_69k_70k.vcf";
-        final String moreSites = getResourceDir() + "bqsr.fakeSitesForTesting.b37.chr17.vcf"; //for testing 2 input files
+        final String HiSeqBam_chr20 = localResources + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam";
+        final String dbSNPb37_chr2021 = largeFileTestDir + "/dbsnp_138.b37.20.21.vcf";
+        final String moreSites = getResourceDir() + "bqsr.fakeSitesForTesting.b37.chr17.vcf"; //for testing 2 input files (FIXME - this file is bogus because uses chr17)
```

Comment:
Shouldn't you fix this as part of this PR? Otherwise how can you claim to resolve https://github.com/broadinstitute/gatk/issues/1017 completely?